### PR TITLE
Payment Request Sample

### PR DESCRIPTION
**Pull request by andypavia**
Adding Payment Request sample....

### CODE REVIEW

**Comment by andypavia** [comment]
@molant @melanierichards

File: paymentrequest/index.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
+<html>
```
Comment (alrra) [inline paymentrequest/index.html]: Add `<!doctype html>` as the first line.

File: paymentrequest/index.html [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+<head>
```
Comment (alrra) [inline paymentrequest/index.html:3]: `<html lang="en-US">`

**Comment by melanierichards** [comment]
fyi @molant I'm going to do some design work on this early next week, so no need to merge this week.

**Comment by molant** [comment]
@andypavia can you please take care of @alrra's comments? I see that @melanierichards wants to take care of some stuff here first so I'll wait for her green light before merging.

**Comment by andypavia** [comment]
@molant I took care of @alrra's comments. We should be good to go after @melanierichards applies her magic.

**Comment by melanierichards** [comment]
@andypavia would you mind granting me push access to your fork? Also, I think the demo needs a short introduction at the top, with a link out to the blog post for further details. Some people may be coming from the demo first, and not have much background on this feature. If you write it and comment here, I can add the intro to the demo myself.

**Comment by andypavia** [comment]
@melanierichards, thanks for the help Melanie. How about the intro below?

The Payment Request API was designed with the goal of standardizing communication across merchants, browsers, and payment methods to provide a better experience for users, and a single, consistent API for developers. 

In Microsoft Edge, the Payment Request API connects to Microsoft Wallet (with the user’s permission), allowing easy access to payment information associated with the user’s Microsoft Account. The wallet can provide a faster and more consistent experience across websites, which saves shoppers time and effort by allowing them to securely share saved payment information. You find more information here. https://blogs.windows.com/msedgedev/2016/12/15/payment-request-api-edge/#5CwWwToEm9YiVdyQ.97

**Comment by melanierichards** [comment]
Closing in favor of #347

**Pull request closed without merging**